[2024-08-30T20:08:53.242+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-30T20:08:58.208+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_dag.get_data manual__2024-08-30T20:07:33.706718+00:00 [queued]>
[2024-08-30T20:08:58.244+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_dag.get_data manual__2024-08-30T20:07:33.706718+00:00 [queued]>
[2024-08-30T20:08:58.247+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-08-30T20:08:58.345+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_data> on 2024-08-30 20:07:33.706718+00:00
[2024-08-30T20:08:58.373+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=1496) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-30T20:08:58.389+0000] {standard_task_runner.py:64} INFO - Started process 1506 to run task
[2024-08-30T20:08:58.379+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'get_data', 'manual__2024-08-30T20:07:33.706718+00:00', '--job-id', '300', '--raw', '--subdir', 'DAGS_FOLDER/envioemail.py', '--cfg-path', '/tmp/tmp0lgybi1i']
[2024-08-30T20:08:58.396+0000] {standard_task_runner.py:91} INFO - Job 300: Subtask get_data
[2024-08-30T20:08:58.590+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_dag.get_data manual__2024-08-30T20:07:33.706718+00:00 [running]> on host 00cc7a82b56e
[2024-08-30T20:08:59.159+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='simple_dag' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-30T20:07:33.706718+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-30T20:07:33.706718+00:00'
[2024-08-30T20:08:59.164+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-30T20:08:59.288+0000] {base.py:84} INFO - Using connection ID 'my_mssql_conn_id' for task execution.
[2024-08-30T20:08:59.623+0000] {warnings.py:112} WARNING - /opt/***/dags/funciones.py:33: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)

[2024-08-30T20:09:03.908+0000] {logging_mixin.py:188} INFO - Datos leidos
[2024-08-30T20:09:03.912+0000] {python.py:237} INFO - Done. Returned value was:         SalesOrderID  OrderQty                ProductID
0              43659         1   Mountain-100 Black, 42
1              43659         3   Mountain-100 Black, 44
2              43659         1   Mountain-100 Black, 48
3              43659         1  Mountain-100 Silver, 38
4              43659         1  Mountain-100 Silver, 42
...              ...       ...                      ...
121312         75122         1    Fender Set - Mountain
121313         75122         1             AWC Logo Cap
121314         75123         1    Fender Set - Mountain
121315         75123         1   All-Purpose Bike Stand
121316         75123         1             AWC Logo Cap

[121317 rows x 3 columns]
[2024-08-30T20:09:04.108+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-30T20:09:05.223+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=get_data, run_id=manual__2024-08-30T20:07:33.706718+00:00, execution_date=20240830T200733, start_date=20240830T200858, end_date=20240830T200905
[2024-08-30T20:09:05.313+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-30T20:09:05.412+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-30T20:09:05.418+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
